Tests/UIViewControllerTests.m

```diff
@@ -52,13 +57,15 @@ - (void)setUp
     _testBundle = [NSBundle bundleForClass:self.class];
     _mainStoryboard = [UIStoryboard storyboardWithName:@"Main" bundle:_testBundle];
     _presentingViewController = [_mainStoryboard instantiateViewControllerWithIdentifier:@"PresentingViewController"];
+    [self.keyWindow setRootViewController: _presentingViewController];
```

Comment:
Was this causing a test failure?

Reply:
Yes. Tests in this file were failing because they expected view lifecycle methods to be called (loadView, viewDidLoad) and they are called only when view is put into current ui stack (displayed)
(The other thing that caused tests to fail was missing code country in current locale on simulator)
The CI job was failing because `--no-ri --no-rdoc` parameters are not supported anymore